test/Interop/Cxx/operators/member-inline-typechecker.swift

```diff
@@ -43,3 +43,9 @@ let diffTypesResultDoubleByVal: Double = diffTypesArrayByVal[0.5]
 
 let postIncrement = HasPostIncrementOperator()
 postIncrement.successor() // expected-error {{value of type 'HasPostIncrementOperator' has no member 'successor'}}
+
+let anotherReturnType = HasPreIncrementOperatorWithAnotherReturnType()
+let anotherReturnTypeResult: HasPreIncrementOperatorWithAnotherReturnType = anotherReturnType.successor()
+
+let voidReturnType = HasPreIncrementOperatorWithVoidReturnType()
+let voidReturnTypeResult: HasPreIncrementOperatorWithVoidReturnType = voidReturnType.successor()
```

Comment:
Maybe a check that the successors .value is indeed 1 more than the predecessor

Reply:
You're right, we should have an execution test for this.